src/plugins/validation/2and3/semantic-validators/operation-ids.js

```diff
@@ -50,15 +54,74 @@ module.exports.validate = function({ resolvedSpec }) {
     return !!prev;
   };
 
+  const operationIdPassedConventionCheck = (opKey, operationId, pathParam) => {
```

Comment:
```suggestion
  const operationIdPassedConventionCheck = (opKey, operationId, hasPathParam) => {
```

This is a little picky but since JS is loosely typed, it's nice when variable names hint at the type of the variable. `pathParam` is a boolean, but sounds like it's a param object. `hasPathParam` makes it clear that the variable is a boolean